event/event.go

```diff
@@ -389,7 +389,7 @@ type TargetFilter struct {
 type Filter struct {
 	Target         Target
 	KindType       kindType
-	KindName       string
+	KindName       []string `form:"-"`
```

Comment:
just a nitpick: maybe we should call this KindName**s**?

Reply:
Agreed!